pom.xml

```diff
@@ -53,6 +53,14 @@
                     </dependency>
                 </dependencies>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>14</source>
+                    <target>14</target>
+                </configuration>
```

Comment:
You don't have to commit all. This can be simply not selected in the changelist.

Reply:
ok :(